y.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
```

Comment:
While I approve of you checking in your test case, maybe you could fold it into Lib/test/test_set.py somewhere?